src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        String[] colors = {"green", "black", "yellow", "white", "purple", "blue", "pink"};
+        Random r = new Random();
```

Comment:
1. random should be `private final` class-level field
2. such short naming is a bad practice, name field as a `random`, not `r` - noone will understand your code if so